zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -19,6 +25,240 @@ use {
     zcash_primitives::legacy::keys::IncomingViewingKey,
 };
 
+struct WalletMigration0;
+
+migration!(
+    WalletMigration0,
+    "bc4f5e57-d600-4b6c-990f-b3538f0bfce1",
+    [],
+    "Initialize the wallet database."
+);
+
+impl RusqliteMigration for WalletMigration0 {
+    type Error = SqliteClientError;
+
+    fn up(&self, transaction: &Transaction) -> Result<(), SqliteClientError> {
```

Comment:
Since I don't know exactly how we use this data, this is an open-ended question.  Are there any indexes that would help with querying for data?  INTEGER PRIMARY KEY and UNIQUE columns are already indexed by default but I'm curious if there are any other columns that we would benefit by indexing?

Reply:
`blocks.hash` should probably be indexed I think.

Reply:
At present we don't ever look up blocks by their hash or join on that column, which is I think why we don't have an index on that column.

Reply:
A unique constraint does make sense though.